src/Components/Dropdowns/DropdownItem.tsx

```diff
@@ -23,6 +24,8 @@ export interface DropdownItemProps extends StandardFunctionProps {
   title?: string
   /** Prevents any interaction with this element, including the onClick function */
   disabled?: boolean
+
+  trailingIconOnSelected?: boolean
```

Comment:
checkmark for when it's selected.

<img width="599" alt="Screen Shot 2022-06-25 at 11 27 13 AM" src="https://user-images.githubusercontent.com/18511823/175786065-cf18fc50-548c-4b57-8504-dff2fe3fc0f1.png">